app/components/page_list_component/_index.scss

```diff
@@ -9,5 +9,26 @@
 }
 
 .app-page-list__key {
-  width: 15%;
+  @include govuk-media-query($from: desktop) {
+    width: 20%;
+  }
+}
+
+.app-page-list__row--error {
```

Comment:
Can we not use the same class that is used for input fields that have errors or maybe use govuk spacing scale too https://design-system.service.gov.uk/styles/spacing/?

Reply:
The padding on the input error class doesn't get on with the summary list's `display: table-row;` declaration on wider screen sizes, but good point about the spacing scale!

Reply:
Actually, we can use the `govuk-form-group--error` class as you suggest - and then add a more targeted fix for the bit where it breaks. Great spot 🙂